src/components/characterList/CharacterList.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import CharacterCard from '../characterCard/CharacterCard';
+import { Character, CharacterResponse } from '../../types/api';
+import { api } from '../../api/api';
+import { LSKEY_PREV_QUERY } from '../../types/constants';
```

Comment:
you can use aliases
https://blog.logrocket.com/using-path-aliases-cleaner-react-typescript-imports/